picasso/src/main/java/com/squareup/picasso/PicassoDrawable.java

```diff
@@ -153,13 +153,13 @@ private void drawDebugIndicator(Canvas canvas) {
   }
 
   private static Path getTrianglePath(Point p1, int width) {
```

Comment:
@JakeWharton yeah, sure. Is it OK if I fix it in another commit, or it will be more correct to create a fresh branch, fix the `getTrianglePath()` signature, its implementation and usage in one commit and make another pull-request?